frodo.js

```diff
@@ -1,12 +1,11 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?